### PR TITLE
[bodhi-updates] manifest: bump to Fedora 31

### PR DESCRIPTION
**Pull request by jlebon**
Following https://github.com/coreos/fedora-coreos-config/pull/200.

### CODE REVIEW

**Comment by dustymabe** [comment]
hmm - there's technically no bodhi runs happening in f31 yet, right? I guess we're just consuming branched content for now?

File: manifest-lock.x86_64.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "packages": {
     "podman": {
-      "evra": "2:1.5.1-3.fc30.x86_64"
+      "evra": "2:1.5.1-3.git0005792.fc31.x86_64"
```
Comment (dustymabe) [inline manifest-lock.x86_64.json]: There is a newer podman in stable?

```
$ koji latest-build f31 podman | tail -n 1
podman-1.6.1-2.fc31
```

Comment (dustymabe) [reply manifest-lock.x86_64.json]: ahh - actually this was a "freeze" because of a regression and I assume a fix hasn't landed yet.

**Review by dustymabe — APPROVED** [review]
LGTM

**Comment by jlebon** [comment]
We should hold this until #200 goes in. Tests won't pass until the changes there are propagated to `bodhi-updates` anyway.

> hmm - there's technically no bodhi runs happening in f31 yet, right?

Oh, I thought it was. Isn't that how we get zero day updates?

**Comment by dustymabe** [comment]
The nightly pungi run is what is controlling the stable repo for F31. There is an updates-testing compose that is running, but no updates composes - you can see it in the list of composes here: https://kojipkgs.fedoraproject.org/compose/updates/ .

Either way it mostly doesn't matter as long as the repo files are pulling in the right content anyway.

**Comment by jlebon** [comment]
OK, this one is ready to go now!

**Review by dustymabe — APPROVED** [review]
LGTM

**Pull request merged**